wordpress-mobile/WordPress-iOS
# Issue/use videopress when available for preview

This PR uses VideoPress URL for video streaming whenever it's available.
It also retrieve VideoPress info from a REST API call instead of relying only in information on the site Media Library.

To test:
 - Start the app
 - Select a WP.com blog that has video saved on it
 - Create a post
 - Tap on Media
 - Select the WordPress.com library
 - Find a video
 - Force Touch/ Long Press
 - Check if the video shows correctly
 - Try the same on a private WP.com
 - Try the same on a jetpack site
 - Try the same on a self-hosted site.

Needs review: @kurzee
